src/Microsoft.ML.Data/Transforms/KeyToVector.cs

```diff
@@ -88,16 +86,26 @@ public sealed class Arguments
             public bool Bag = KeyToVectorMappingEstimator.Defaults.Bag;
         }
 
-        public class ColumnInfo
+        /// <summary>
+        /// Describes how the transformer handles one column pair.
+        /// </summary>
+        public sealed class ColumnInfo
         {
             public readonly string Input;
             public readonly string Output;
             public readonly bool Bag;
 
-            public ColumnInfo(string input, string output, bool bag = KeyToVectorMappingEstimator.Defaults.Bag)
+            /// <summary>
+            /// Describes how the transformer handles one column pair.
+            /// </summary>
+            /// <param name="input">Name of input column.</param>
```

Comment:
Would it be useful to specify the acceptable input data type(s)? Given the name, I assume only `Key` type is acceptable.